test/e2e/testdata/acme/clusterissuer_gcp.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: acme-dns01-clouddns-ambient
+spec:
+  acme:
+    preferredChain: ""
+    privateKeySecretRef:
+      name: issuer-acme-letsencrypt
+    server: https://acme-staging-v02.api.letsencrypt.org/directory
+    solvers:
+    - dns01:
+        cloudDNS:
+          # The ID of the GCP project
+          project: OPENSHIFT-QE
```

It is better to use general placeholder text instead of concrete ID value.
```suggestion
          project: PROJECT_ID
```

---

+1

---

@geliu2016 it is better to use general `project: PROJECT_ID` instead of either `project: OPENSHIFT_QE` or `project: OPENSHIFT-QE`

---

updated.